include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -369,14 +423,80 @@ struct touch : public base_turn_handler
         return side1 == side2 && ! opposite(side1, turn);
     }
 
-    /*static inline void block_second(bool block, TurnInfo& ti)
+#if ! defined(BOOST_GEOMETRY_USE_RESCALING)
+    template
+    <
+        typename UniqueSubRange1,
+        typename UniqueSubRange2
+    >
+    static inline bool handle_imperfect_touch(UniqueSubRange1 const& range_p,
+            UniqueSubRange2 const& range_q, TurnInfo& ti)
```

Comment:
indentation issue?

Reply:
Well, this is how we (at least I) did it from the beginning - an indentation of 8. However, currently IDE's usually automatically align it under the next bracket and some features (`clang_format`) or people also do this. So now we've a mixed approach.

IMO both are not perfect... For the first method it is currently unusual, I think. For the second method, with long names and modifiers, it goes really to the right and needs more lines.

I keep this for now, if you've strong opinions, just comment.